client/lcd/swagger-ui/swagger.yaml

```
@@ -40,7 +40,7 @@ paths:
       description: Information about the connected node
       summary: The properties of the connected node
       tags:
-        - Tendermint RPC
+        - Gaia REST
```

this is just the tag under which the endpoints are grouped
```suggestion
        - Tendermint RPC
```